components/inputs/test/input-time.test.js

```diff
@@ -0,0 +1,130 @@
+import '../input-time.js';
+import { aTimeout, expect, fixture, oneEvent } from '@open-wc/testing';
+
+const basicFixture = '<d2l-input-time label="label text"></d2l-input-time>';
+
+describe('d2l-input-time', () => {
+
+	let elem;
+
+	beforeEach(async() => {
+		elem = await fixture(basicFixture);
```

Comment:
Now that it's super easy to create fixtures in the open-wc stuff, I'm actually more a fan of not reusing the same fixture for everything and manipulating in JS. Like have one default fixture, but then when you need something slightly different, create it, like this:

```
it('passes all axe tests when label is hidden', async() => {
    const elem = await fixture(html`<d2l-input-time label="label" label-hidden></d2l-input-time>`);
    await expect(elem).to.be.accessible();
});
```

There are several cases down below where you're using JS to manipulate state before asserting things where you could just use a different fixture initially.